lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/ChukwumaIbezimTest/BankAccountTest.java

```diff
@@ -0,0 +1,74 @@
+package com.codedifferently.lesson7.ChukwumaIbezimTest;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.codedifferently.lesson7.chukwumaibezim.*;
+import com.codedifferently.lesson7.chukwumaibezim.BankAccount.InsufficientFundsException;
+import org.junit.jupiter.api.Test;
+
+public class BankAccountTest {
```

Comment:
Last thing. Your test should mirror the same package and class as the file under test.

Reply:
updated